### PR TITLE
QAE-882 - E2E test case for button text and background color option

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Header-builder → Transparent Header → Design → button text and background color

### Screenshots

https://share.getcloudapp.com/RBunzNke

### How has this been tested?

npm run test:e2e:interactive -- tests/e2e/specs/customizer/header-builder/transparent-header/button-text-bg-color.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/header-builder/transparent-header/button-text-bg-color.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+						0: 'button-1',
+					},
+				},
+			},
+		};
+		await setCustomize( buttonBackgroundColor );
+		await page.goto( createURL( '/' ), {
+			waitUntil: 'networkidle0',
+		} );
+		await page.waitForSelector( '.ast-theme-transparent-header [CLASS*="ast-header-button-"] .ast-custom-button' );
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/transparent-header/button-text-bg-color.test.js:43]: @SimranChougule1 test case is failing here, please execute at the local end

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/transparent-header/button-text-bg-color.test.js:45]: Resolved

File: tests/e2e/specs/customizer/header-builder/transparent-header/button-text-bg-color.test.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import { createURL } from '@wordpress/e2e-test-utils';
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Transparent header settings in the customizer', () => {
+	it( 'button text color setting should apply correctly', async () => {
+		const buttonTextColor = {
+			'transparent-header-enable': 1,
+			'transparent-header-disable-latest-posts-index': 0,
+			'transparent-header-button-text-color': 'rgb(70, 5, 95)',
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/header-builder/transparent-header/button-text-bg-color.test.js:8]: @SimranChougule1 it showing color-contrast error,could you please change the color

Comment (SimranChougule1) [reply tests/e2e/specs/customizer/header-builder/transparent-header/button-text-bg-color.test.js]: changed colors

**Review by GauravKhupse — APPROVED** [review]

**Review by bsfsnehal — APPROVED** [review]

**Pull request merged**